shiptest.dme

```diff
@@ -14,6 +14,7 @@ g// DM Environment file for shiptest.dme.
 
 // BEGIN_INCLUDE
 #include "_maps\_basemap.dm"
+#include "_maps\RandomRuins\JungleRuins\jungle_botany.dmm"
```

Comment:
Dont tick your map file.
Instead add a ruin datum for your ruin.

Reply:
Got it, let me change that

Reply:
Where do I add the datum?